tests/run/test_grammar.py

```diff
@@ -487,7 +486,6 @@ class Cbad2(C):
                 x: int
                 x.y: list = []
 
-    @skip("Class annotations not implemented")
     def test_var_annot_metaclass_semantics(self):
         class CMeta(type):
             @classmethod
```

Comment:
@scoder it appears that CPython supports custom namespaces for the annotations dict - how would we proceed with this in the compiler? Or is this a feature we should punt on?

Reply:
Hmm. Well, to implement this correctly, we'd really have to initialise `"__annotations__"` at the start, and then look it up each time we see an annotated variable in the code and update whatever mapping we have then. That's a bit more complex than what we have now (although it would certainly use the same infrastructure).

I'd be happy to get the main feature working and merged, and then look at more details later. This can be changed incrementally, if someone feels like doing it.

Reply:
Sounds good - I'll re-add the skip with an updated comment and create a ticket for the work.

Reply:
Covered in #3839